examples/create-next-app/pages/index.js

```diff
@@ -0,0 +1,46 @@
+import Head from 'next/head'
+import { Text, Card, Note, Code, Spacer } from '@zeit-ui/react'
+
+export default function Home() {
+  return (
+    <div className="container">
```

Comment:
Maybe we can try `Page` component here?

Reply:
Sure, that was created by create-next-app, just pushed a commit with this change.